aws/cloudformation/iam.yml.erb

```diff
@@ -3,6 +3,10 @@ AWSTemplateFormatVersion: 2010-09-09
 Description: >-
   IAM layer including global/shared roles and access permissions for Code.org infrastructure.
   Note: Admin permissions are required to manage some admin-only resources in this stack.
+Parameters:
+  GoogleDeveloperIDs:
+    Type: AWS::SSM::Parameter::Value<List<String>>
```

Comment:
Is the new process for onboarding to update this list in the Systems Manager console and then run an update for this stack?

Reply:
yeah- there is a potential path to automate this further, (e.g., a Lambda 'syncGoogleFederation' function, that's provided read-only Google admin-directory and developer-role stack-update credentials, that periodically lists 'developer' users in the Google account, updates this parameter with the result, and then updates the role-containing stack) but for now it's a two-step manual process as you describe.